generators/client/templates/common/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "devDependencies": {
+    "dayjs": "1.9.3",
```

Comment:
Should add to dependencies instead of devDependencies.

Reply:
Oops, sorry

Reply:
It's fixed